libnavigation-util/src/main/java/com/mapbox/navigation/utils/extensions/AsyncUtil.kt

```diff
@@ -0,0 +1,10 @@
+package com.mapbox.navigation.utils.extensions
+
+import kotlinx.coroutines.CoroutineScope
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.async
+import kotlinx.coroutines.runBlocking
+
+fun <A, B> List<A>.pmap(f: (A) -> B, scope: CoroutineScope): List<B> = runBlocking {
```

Comment:
It would be great if the extension name was a little bit more descriptive, plus a KDoc.